hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpProperties.java

```diff
@@ -39,6 +41,11 @@
     private boolean missingQueuesFatal = false;
 
     /**
+     * Requested heartbeat interval from broker in {@link TimeUnit#SECONDS}.
+     */
+    private int requestedHeartBeat = 60;
```

Comment:
Is it better to use `(int) TimeUnit.SECONDS.toSeconds(60)`. We use this in some other configurations